apps/heft/src/templates/copy.json

```diff
@@ -0,0 +1,108 @@
+/**
+ * Configures the "copy" task for Heft.
+ */
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/heft/copy.schema.json",
```

Comment:
@iclanton  File copying seems like a "core operation" that belongs in **heft.json** rather than in a plugin-like config file.  Another "core operation" would be executing simple shell scripts.  Both of these actions could be triggered based on the same menu of Heft events.

Thus I'm thinking the config file design might be better like this:

**config/heft.json**
```js
"eventActions": {
  "beforePreCompile": [
    {
      "eventActionKind": "fileCopy",
      "sourceGlob": "src/**/*.md",
      "targetFolders": [ "dist" ],
      "hardlinkInsteadOfCopy": true
    }
  ],
  "afterPostBuild": [
    {
      "eventActionKind": "shell", // <-- to be introduced in a future PR
      "command": "rundown",
      "args": [
        "snapshot",
        "--script",
        "./lib/index.js"
      ]
    }
  ]
}
```